src/main/java/com/agnacore/spaceship/game/Ship.java

```diff
@@ -9,18 +9,21 @@ public class Ship {
 	private int health, speed;
 	private int[] position;
 	private int[] offset;
+	private String image;
 	
-	public Ship(int h, int s, int x, int y, int width, int height) {
+	public Ship(int h, int s, int x, int y, int width, int height, String img) {
 		this.health = h;
 		this.speed = s;
 		this.position = new int[2];
 		this.position[0] = x;
 		this.position[1] = y;
+		this.image = img;
 		
 		// the offset is how far from the middle (position) the borders are
 		// this makes for easy collision detection...
 		// with the downside of the actual width or height being 1 pixel larger if they were even
 		// I could fix that, but it sounds like more effort than it's worth lol
+		// update: never mind, I want to fix this now
```

Comment:
If you begin a comment with `TODO`, your IDE should highlight it to draw more attention.
![todo](https://github.com/agnacore/Spaceship/assets/42688474/52d4f236-7020-4aa3-bff6-93d970e79187)